google/cloud/forseti/common/util/regular_exp.py

```diff
@@ -41,4 +43,8 @@ def escape_and_globify(pattern_string):
     # pylint: enable=anomalous-backslash-in-string
     if pattern_string == '*':
         return '^.*$'
-    return '^{}$'.format(re.escape(pattern_string).replace('\\*', '.+?'))
+
+    # TODO(2189): Make wildcards always 0 or more.
+    replace = '.*' if wildcard_is_zero_or_more else '.+'
```

Comment:
Do we need the '?' at the end? e.g. '.*?' and '.+?' (Looks like it was there from before)

Reply:
I think it's a no-op. Do you know if an example where it did anything?